### ISSUE TITLE
Ignore *all* duplicates, please

### ISSUE THREAD

**Issue by Himmelmeister** [markusfisch/BinaryEye#439]
Hello!

I'm using Binary Eye to scan multiple barcodes from packages of new devices, and transfer them to my PC
I need multiple barcodes, but want every barcode only once.

The problem is that Binary Eye only suppresses double scans if they are performed directly following each other.
However, if three or more barcodes are in the field of view or are scanned one after the other, duplicates are no longer filtered.

Basically, it is sufficient for three barcodes to be in front of the camera at the same time. Binary Eye then scans all three in an endless loop and fills the scan list to infinity.

**I would therefore like to have an option to only transfer barcodes to the history if they are not already there.**

I hope that I have been able to express my problem understandably. :)

Beside that, Binary Eye is great, thank you!

**Comment by markusfisch** [comment]
Ah, I see!

And yes, it's very much possible to ignore all duplicates! I will add an option for that with the next version.

**Comment by markusfisch** [comment]
Implemented in https://github.com/markusfisch/BinaryEye/commit/69baa3b82410a4e7a0e1a9288b3bdf754f94324d

A new version will become available soon.
